packages/react/src/components/button/__snapshots__/Button.test.tsx.snap

```diff
@@ -14,3 +14,26 @@ exports[`<Button /> spec renders the component 1`] = `
   </button>
 </DocumentFragment>
 `;
+
+exports[`<Button /> spec renders the loading button component 1`] = `
+<DocumentFragment>
+  <button
+    aria-disabled="true"
+    class="button primary theme-default size-default isLoading"
+    type="button"
+  >
+    <div
+      class="loadingSpinner small"
+    >
+      <div />
+      <div />
```

Comment:
Hmm.. 3 empty divs here 🤔

Reply:
This is the html generated by LoadingSpinner component